src/asm/ram32.s

```diff
@@ -0,0 +1,60 @@
+.section .ram32, "ax"
+.code32
+
+ram32_start:
+    # Indicate (via serial) that we are executing out of RAM
+    movw $0x3f8, %dx
+    movb $'R', %al
+    outb %al, %dx
```

Comment:
Should we keep these characters being printed to serial output? I found it useful when debugging/writing the assembly, but it is a little weird to see `RPL` printed right before the firmware starts up.

Reply:
Fine for now, but maybe we could look into splitting out output between 0x402 (QEMU firmware I/O debug port) and the serial port. This could be a candidate for that.

Reply:
I removed all the serial logs from the assembly. I just have them in a separate branch for local debugging.